lib/prmd/templates/schemata/helper.erb

```diff
@@ -137,8 +137,11 @@
     end
 
     if value['pattern']
-      # Prevent the pipe regex pattern characters from being interpreted as markdown table:
-      pattern = value['pattern'].gsub(/\|/, '&#x7c;')
+      # Prevent the pipe regex pattern characters from being interpreted
+      # as markdown table
+      # Prevent instances of []() in regexes from being rendered as
+      # markdown links
+      pattern = value['pattern'].gsub(/\|/, '&#124;').gsub(/\[/, '&#91;')
```

Comment:
Why the change from `&#x7c;` to `&#124;`? It looks like 124 is the right HTML entity from brief web searching, so I'm not saying this is wrong, just wanted to confirm. Also, if this is using 124, the test below should also use it. I think being consistent between the two places is important, even if I'm more indifferent about which of the two we choose (presuming they both render in the desired way).

Reply:
Left a comment on this on the main conversation for better visibility